octopoes/tests/test_query.py

```diff
@@ -372,3 +372,7 @@ def test_build_parth_query_with_multiple_sources_for_abstract_type(mocker):
         ' (or [ IPAddress :object_type "IPAddressV4" ] [ IPAddress :object_type "IPAddressV6" ] )'
         ' [ Network :object_type "Network" ]]}}'
     )
+
+
+def test_parse_path_concrete_fields_or_abstract_types():
+    assert len(Path.parse("URL.web_url.netloc.name").segments) == 3
```

Comment:
It proves that it doesn't fail, as it did before, and it proofs that it gets the right amount of segments for these new kinds of path queries (=amount of dots)?

Reply:
Sure it's a good thing to know that it fixed something, but still doesn't prove enough. Usually when testing a parser, you'd be more interested in that the parser actually works, rather than only count the number of parsed elements. Something like `assert ...segments = [ ..., ..., ... ]` would prove that the parser works as expected and that it successfully parsed 3 elements.

Reply:
Yeah fair enough hehe